### PR TITLE
Feature/write tests

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Feature/write tests

In this Upate:

- [ ] Write unit tests for Author
- [ ] Write tests for game

### CODE REVIEW

File: modules/game_module.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  end
+
+  def save_games
+    data = []
+    puts @games.to_s
+    @games.each do |game|
+      data.push({ multiplayer: game.multiplayer, last_played_at: game.last_played_at,
+                  published_date: game.publish_date.to_s })
+    end
+    File.write('./json/games.json', JSON.generate(data))
```
Comment (topeogunleye) [inline modules/game_module.rb]: - [x] [REQUIRED] There is no json folder in your codebase but you are specifying './json' to save your game data, so your code will name be able to save. Kindly fix this please

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @blessedjasonmwanza,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * You documented your work professionally. :heavy_check_mark:
> * Your linters test pass :heavy_check_mark: .
> 
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check the replies to your inline comments.
> * [ ] Please your tests do not pass.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding:!clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: modules/game_module.rb (hunk 1 above)
Comment (blessedjasonmwanza) [reply modules/game_module.rb]: Fixed

Comment (blessedjasonmwanza) [reply modules/game_module.rb]: Kindly note: that I am still working on this PR. I will request for a review once done.

**Review by afizsavage — APPROVED** [review]
### Hi @blessedjasonmwanza 
congratulations

Comments: Approval
# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**